WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -102,40 +102,11 @@ class CommentDetailViewController: UIViewController {
         return cell
     }()
 
-    private lazy var deleteButton: UIButton = {
-        let button = UIButton()
-        let buttonColor = UIColor(light: .error, dark: .muriel(name: .red, .shade40))
-        button.translatesAutoresizingMaskIntoConstraints = false
-        button.setTitle(.deleteButtonText, for: .normal)
-        button.setTitleColor(buttonColor, for: .normal)
-        button.setTitleColor(.white, for: .highlighted)
-        button.setBackgroundImage(UIImage.renderBackgroundImage(fill: .clear, border: buttonColor), for: .normal)
-        button.setBackgroundImage(.renderBackgroundImage(fill: buttonColor, border: buttonColor), for: .highlighted)
-
-        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .semibold)
-        button.titleLabel?.textAlignment = .center
-        button.titleLabel?.numberOfLines = 0
-
-        // add constraints to the title label, so the button can contain it properly in multi-line cases.
-        if let label = button.titleLabel {
-            button.pinSubviewToAllEdgeMargins(label)
-        }
-
-        button.on(.touchUpInside) { [weak self] _ in
-            self?.deleteButtonTapped()
-        }
-
-        return button
-    }()
-
-    private lazy var deleteButtonCell: UITableViewCell = {
-        let cell = UITableViewCell()
-        cell.selectionStyle = .none
-        cell.accessibilityTraits = .button
-
-        cell.contentView.addSubview(deleteButton)
-        cell.contentView.pinSubviewToAllEdges(deleteButton, insets: Constants.deleteButtonInsets)
```

Comment:
I think `Constants.deleteButtonInsets` [defined here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17505/files#diff-fb74f65f40fd00682ceebb579df4be3d45796af24b8cf138ef0ba5915a44b8cbL268) could be deleted since it's no longer used.

Reply:
Woops forgot about that. Thanks! And done!